tests/codebase/test_license.py

```diff
@@ -0,0 +1,37 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2021, Anaconda, Inc. All rights reserved.
+#
+# Powered by the Bokeh Development Team.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+import pytest ; pytest
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+from os import chdir
+from os.path import join
+from subprocess import run
+
+# Bokeh imports
+from . import TOP_PATH
+
+#-----------------------------------------------------------------------------
+# Tests
+#-----------------------------------------------------------------------------
+
+def test_license_set() -> None:
+    ''' Ensure the top-level repo LICENSES.txt always matches the copy in
+    the Python package folder (needed there when gnerated packages).
```

Comment:
generating

Reply:
FYI @mattpap if possible for small changes like this,  just hitting the "suggestion" button and making the comment a suggestion makes it much faster/streamlined to just commit the fix right here